plothist/hep_plotters.py

```diff
@@ -145,7 +145,7 @@ def compare_data_mc(
             uncertainties_high**2,
         )
         data_hist[:] = np.stack([data_hist.values(), data_variances], axis=-1)
-    elif comparison_kwargs["comparison"] == "ratio":
+    elif comparison_kwargs["comparison"] in ["ratio", "relative_difference"]:
```

Comment:
After all the `elif`, we should add an `else` that raises a `NotImplementedError` to avoid the same kind of bugs in the future if we add new comparisons